tough/src/target_name.rs

```diff
@@ -137,10 +137,10 @@ fn clean_name(name: &str) -> Result<String> {
                 .context(error::TargetNameComponentsEmptySnafu { name })?
                 .as_os_str();
 
-            // If the first component isn't `/` then there is a bug or behavior change in
+            // If the first component isn't the main separator ( unix `/`, windows '\' ) then there is a bug or behavior change in
             // absolutize_from.
```

Comment:
Nit: we wrap comments at 100-char width.

```suggestion
            // If the first component isn't the main separator ( unix `/`, windows '\' ) then there
            // is a bug or behavior change in absolutize_from.
```

Reply:
I've updated that. I thought that would be handled by rustfmt.